### ISSUE TITLE
Logs pulling from iOS device

### ISSUE THREAD

**Issue by sskorol** [isonic1/appium-mobile-grid#3]
Hi @isonic1,

I see you're using `logcat` for pulling logs from Android devices. But what about iOS? Corresponding helper is empty. Which tool are you using for that?

Thanks,
Sergey

**Comment by sskorol** [comment]
Just checked `flick` and seems like `idevicesyslog` is used for that. But is there any way for logs' cleanup before test execution on iOS? With `logcat` it could be done via `-c` flag.

**Comment by isonic1** [comment]
hmm, that is a good question. I'm not sure about ios. However, once your test starts idevicesyslog should start capturing the data at the point and not before. Are you seeing otherwise?

**Comment by sskorol** [comment]
@isonic1 thanks for your thoughts. Will check it in ~3 days and let you know.

**Comment by sskorol** [comment]
@isonic1 sorry, forgot to leave my feedback on this one.
Unfortunately, `idevicesyslog` grabs data for earlier period as well. I got results for the past hour apart from recorded time.

**Comment by isonic1** [comment]
Yeah, It's something with idevicesyslog which I cannot control. However, I
add an enhancement in Flick to insert a test start point in the log.


https://github.com/isonic1/flick/issues/15
